run.sh

```diff
@@ -2,4 +2,50 @@
 # Dev Server
 #. venv/bin/activate && python manage.py runserver
 # Prod Server
-. venv/bin/activate && gunicorn -b 0.0.0.0:8000 MobSF.wsgi:application --workers=1 --threads=10 --timeout=1800
\ No newline at end of file
+var="$1"
+
+function validate_ip () {
+local IP=$1
+local stat=1
+
+if [[ $IP =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then
```

Comment:
This is nice!! Can windows batchfile support any validations like this?

Reply:
need to check for windows :)